code/game/objects/items/holosign_creator.dm

```diff
@@ -13,6 +13,7 @@
 	throw_range = 7
 	item_flags = NOBLUDGEON
 	var/list/signs = list()
+	var/list/holodesigns = list()
```

Comment:
Move this to the multi subtype

Reply:
Looks like this has been modified. Doing a final check before merge